roles/infra/tasks/main.yml

```diff
@@ -46,6 +46,12 @@
     content: "{{ _terraform_backend_config | to_json }}"
     dest: "{{ terraform_project_path }}/backend_config.json"
 
+- name: Set terraform to rebuild patched appliances
+  include_tasks: rebuild.yml
+  when:
+    - terraform_rebuild_nodes
```

Comment:
We should set a role default that sets this to `false` so that we don't get undefined variable warnings when it isn't supplied, e.g. if the appliance doesn't support it yet.